github/activity_star.go

```diff
@@ -71,12 +73,16 @@ type ActivityListStarredOptions struct {
 // GitHub API docs: https://docs.github.com/en/rest/activity/starring#list-repositories-starred-by-a-user
 func (s *ActivityService) ListStarred(ctx context.Context, user string, opts *ActivityListStarredOptions) ([]*StarredRepository, *Response, error) {
 	var u string
+	var err error
 	if user != "" {
-		u = fmt.Sprintf("users/%v/starred", user)
+		u, err = newURLString("users/%v/starred", user)
 	} else {
-		u = "user/starred"
+		u, err = newURLString("user/starred")
```

Comment:
This line can stay as it was before. Any time `newURLString` has only a single constant arg, that change can be reverted.